### DISCUSSION TITLE
Reference to v2.5.13?

### DISCUSSION THREAD

**Discussion by mechatroniks-git** [SpenceKonde/megaTinyCore#757] [Q&A]
Seeing a reference to a version that doesn't exist, is that correct?

See: https://github.com/SpenceKonde/megaTinyCore/blob/master/megaavr/extras/Ref_Tuning.md

**Comment by ObviousInRetrospect** [comment]
was added here: https://github.com/SpenceKonde/megaTinyCore/commit/702fd19242930af09edf0f02b61d802d46dfa95a

almost certainly should be 2.5.12

**Reply by SpenceKonde** [reply]
Naw, I should be 2.6.0
